Adding aggregated roots config. (#19937)
ckeditor/ckeditor5: .changelog/20260316165921_ck_19885.md

---
type: Other
scope:
  - ckeditor5-core
  - ckeditor5-editor-classic
  - ckeditor5-editor-inline
  - ckeditor5-editor-balloon
  - ckeditor5-editor-decoupled
  - ckeditor5-editor-multi-root
  - ckeditor5-watchdog
closes:
  - https://github.com/ckeditor/ckeditor5/issues/19885
---

Introduced a root-related configuration aggregation in the editor configuration.
The `config.initialData`, `config.placeholder`, and `config.label` options from the editor configuration have been deprecated 
and replaced with the `config.root.initialData`, `config.root.placeholder`, `config.root.label` options. 

> Additionally, for multi-root editor the `config.roots` option has been introduced, so the above options can be defined for each root

Reviewer: ```suggestion
For multi-root editor the `config.roots` option has been introduced. The above options should be now defined for each root
```
